modules/apps/object/object-test/src/testFunctional/macros/ObjectField.macro

```diff
@@ -22,6 +22,35 @@ definition {
 			value1 = ${folderName});
 	}
 
+	macro addTranslatableTextEntryViaUI {
+		LexiconEntry.gotoAdd();
+
+		Type(
+			key_fieldLabel = ${fieldLabel},
+			locator1 = "TextInput#GENERIC_TEXT_INPUT",
+			value1 = ${mainEntryValue});
+
+		Click(
+			key_buttonFieldEntryLanguage = ${buttonFieldEntryLanguage},
+			locator1 = "ObjectField#BUTTON_FIELD_ENTRY_LANGUAGE");
+
+		Click(
+			key_fieldEntryLanguage = ${fieldEntryLanguage},
+			locator1 = "ObjectField#SELECT_FIELD_ENTRY_LANGUAGE");
```

Comment:
Probably not serve @patriciajeanperez since the `PageEditor.changeLocale` only select the language but not type the value inside the value. Besides that the macro contains the `waitForForm` macro is focused in another component, not Objects.

Reply:
I analyzed and just one works for the context